subsys/net/ip/Kconfig.debug

```diff
@@ -91,7 +91,16 @@ config NET_DEBUG_NET_PKT
 	default n
 	select NET_BUF_POOL_USAGE
 	help
-	Enables printing of network packet and buffer allocations and frees.
+	Enables collection of network packet and buffer allocations and frees.
+
+config NET_DEBUG_NET_PKT_ALL
+	bool "Debug network packet and buffer individual allocation"
```

Comment:
That reminds me I actually ask for better naming (though indeed makes the patch more complicated). NET_DEBUG_NET_PKT could be kept, and would be the one printing out everything. But an additional NET_DEBUG_NET_PKT_MEM_STATS (for instance) could be introduced and would act as collecting net pkt/buf allocs and frees.